content/applications/sales/sales/amazon_connector/setup.rst

```diff
@@ -119,41 +77,39 @@ the subtotals between Seller Central and Odoo.
 Add an unsupported marketplace
 ==============================
 
-Some Amazon Marketplaces, such as Amazon Brazil, are not officially supported by Odoo but might be
+Some Amazon Marketplaces, such as Amazon Brazil, are not created by default by Odoo but might be
```

Comment:
I believe it's important to state that those marketplaces at not supported. Maybe combine both statements?

Reply:
They should work, that's why on the application in the Seller Central, we put that every marketplace are supported, @dalaOdoo has a Japanese contact who will test it I think (well, we talked about it at least, I don't know if the customer is still interested). If it works in Japan, it'll work everywhere, and we can say they are supported.

Reply:
After discussing it, this'll be changed in master